docs/dotnet-configuration.md

```diff
@@ -4,6 +4,91 @@ title: Scope .NET Agent configuration
 sidebar_label: Configuration
 ---
 
+The behaviour of the Scope Agent can be modified by using  environment variables or by using a configuration file `Scope.yml`in the source root of the project.
+
+> These configuration settings can also be provided via CLI flags. Run `scope-run --help` for more information.
+
+
+
+## Disabling Scope instrumentation
+
+You can disable Scope auto instrumentation. Scope auto instruments your code and tests by default, you can disable this behaviour.
+
+### Using Environment Variables
+
+```sh
+SCOPE_AUTO_INSTRUMENT=False
+```
+
+### Using YAML Configuration file
+
+```yaml
+scope:
+  auto_instrument: false
+  
+```
+
+
+
+## Changing service name
+
+You can change the name of the service when sending data to Scope. Scope initially uses `default` for the service name.
+
+### Using Environment Variables
+
+```sh
+SCOPE_SERVICE="service name"
```

Comment:
same feedback as before and for all the rest

Reply:
Changed to "service-name"